articles/openstack/ostack-how-basic-heat-template.md

```diff
@@ -0,0 +1,164 @@
+---
+title: How to create and deploy a basic heat template  | UKCloud Ltd
+description: Instructions on how to deploy a basic heat template
+services: openstack
+author: Bryce Nicholls
+toc_rootlink: How to
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create and deploy a basic heat template 
+toc_fullpath: How To/ostack-how-static-ip.md
+toc_mdlink: ostack-how-basic-heat-template.md
+---
+
+# How to create and deploy a basic heat template 
+
+## Overview
+
+This is a basic heat orchestration template create a heat stack.
```

Comment:
Suggest:
This articles provides a basic Heat Orchestration template that creates a Heat stack.